tsconfig.json

```diff
@@ -12,91 +12,101 @@
     ],
     "target": "es6",
     "module": "commonjs",
-    "baseUrl": "src/main",
+    "baseUrl": "src" ,
     "paths": {
-      "app/*": [
-        "app/*"
-      ],
+
       "payment-hub-client/*": [
-        "app/pay/*"
+        "main/app/pay/*"
       ],
       "client/*": [
-        "app/client/*"
+        "main/app/client/*"
       ],
       "postcode-lookup/*": [
-        "app/postcode-lookup/*"
+        "main/app/postcode-lookup/*"
       ],
       "drafts/*": [
-        "app/drafts/*"
+        "main/app/drafts/*"
       ],
       "claims/*": [
-        "app/claims/*"
+        "main/app/claims/*"
+      ],
+      "app-common/*": [
```

Comment:
what do you think about using either `common` or `shared` instead of `app-common` I'm not a big fan of the combined name?

Reply:
`common` would have conflicts with another alias, but `shared` would certainly be a better name.

Reply:
Happy with `shared` to start with. In the future we would like to remove app directory so the need for having both `common` and `shared` might disappear.